Jenkinsfile

```diff
@@ -24,7 +24,9 @@ node {
             export const apiHost = \'${env.API_HOST}\';
             export const defaultScenario = \'RCP85\';\n'''
 
-        sh 'scripts/cibuild.sh'
+        sh 'scripts/setup'
```

Comment:
The [Jenkinsfile best practices](https://www.cloudbees.com/blog/top-10-best-practices-jenkins-pipeline-plugin) suggest keeping each unit of work in its own`stage` block to visually represent the steps of the build. I think we'd want `setup`, `test`, `cibuild` and `cipublish` blocks in this project.